docs/DISTRIBUTION.md

```diff
@@ -128,12 +128,50 @@ Each run creates all hourly and daily archives for the last 14 days. To prevent
 
 ## Diagnosis Key Export Files
 
-Coming soon
+At the end of the process the diagnosis keys are exported into ZIP archives that contain two entries:
+
+- export.bin - the binary containing the exposure keys
+- export.sig - the signature to verify the export binary
+
+The `export.bin` file of the archive contains the Temporary Exposure Keys that are broadcasted by the devices of the
+people who are diagnosed with COVID-19.
+The `export.sig` file is used by the server to sign each `export.bin` with a private key. The `export.sig` file
+contains the raw signature as well as additional information needed for verification.
 
 ## Signing
 
-Coming soon
+The signing of the files basically means creating and signing an archive that contain the `export.bin` file
+(the file name is configurable in `application.yaml` under `tek-export.file-name` property).
+Both the application configuration as well as the Temporary Exposure Keys are signed so that the mobile devices
+can verify if the files are originating from the right backend. Using the signature, attack vectors like a
+man in the middle attack, request forgery etc. can be mitigated successfully. The implementation is done in
+[`AppConfigurationSigningDecorator`](/services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/structure/archive/decorator/signing/AppConfigurationSigningDecorator.java).
+
+The signing process takes the `export.bin` archive and creates a signature of the data structure which is then written into the `export.sig`.
+Please see`getSignatureFile` method located in [`SigningDecoratorOnDisk`](/services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/archive/decorator/signing/SigningDecoratorOnDisk.java).
+There are different implementations for that, i.e. how to sign an app configuration which is the
```

Comment:
Because the export.bin archive contains different information (i.e. keys, app configuration), there are also different  implementations for signing each type of data structure:
- AppConfigurationSigningDecorator [link] / explanation
- DiagnosisKeySigningDecorator [link] / explanation
- TestSigningDecorator -> not sure exactly when this is used, or if used in prod environments

Reply:
The TestSigningDecorator it's used only in tests.